docs/intro-react.md

```diff
@@ -56,14 +56,16 @@ Your component starts as a function:
 function Cat() {}
 ```
 
-Whatever a function component returns is rendered as a React element. `Cat` will render a `<Text>` element:
+Whatever a function component returns is rendered as a **React element.** `Cat` will render a `<Text>` element:
 
 ```jsx
 function Cat() {
   return <Text>Hello, I am your cat!</Text>;
 }
 ```
 
+Here, `Cat` is the component, a blueprint. What it returns, `<Text>`, is a React element, an object that describes the Views that the compnent represents.
```

Comment:
>Here, `Cat` is the component, a blueprint.

I like this.

>What it returns, `<Text>`, is a React element, an object that describes the Views that the compnent represents.

I think this sounds a bit too abstract — there's multiple levels of indirection ("element, an object that describes views that component represents"). If you don't know these terms already I think it's too dense.

Here's how I would word it:

>Here, `Cat` is the component, a blueprint. What it returns, `<Text>`, is a React element. React elements let you describe what you want to see on the screen.

Reply:
Hmmm... I want to reinforce what was said in the previous section: that Native Components are backed by Views. Where we might say on the Web

> a React element describes the DOM nodes the component represents 

in React Native mightn't we say 

> a React element describes the Views the component represents

<img width="489" alt="pedantic-raptor" src="https://user-images.githubusercontent.com/236306/76323843-2833cf80-62dd-11ea-9256-5114a2f06146.png">